### PR TITLE
update docker documentation

### PR DESCRIPTION
**Pull request by hmunfru**
#### REVIEWERS

@flopezag 
#### DESCRIPTION
- Update documentation with docker information.
#### TESTING

Only documentation

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -88,6 +88,16 @@ The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor
 same, but with 0GB of disk (i.e. a minimal disk to boot the image is created
 but with barely free space)
 
+Deploying three testbeds
+************************
+
+The *deploythreeglances.py* script deploys three Openstack Image Repository service (Glance) which
```
Comment (flopezag) [inline README.rst]: 3 services

File: README.rst [hunk 2]
```diff
@@ -88,6 +88,16 @@ The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor
...
+shares the same keystone. Each glance is deploy in a different VMs accesible by floating IPs.
```
Comment (flopezag) [inline README.rst]: which share (they are 3 glances, aren't?)

each glance is deployed in a different VM accesible via floating IP.

Please be homogeneous, if you start using server continue with server and do not mix with VM or virtual machine. Server is the concept used in OpenStack and I prefer to use it.

File: README.rst [hunk 3]
```diff
@@ -88,6 +88,16 @@ The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor
...
+As well as previoly, it is possible to access by SSH Key at *~/.ssh/testbedskuld_key*). Usually the installation
```
Comment (flopezag) [inline README.rst]: I suggest to rewrite from here until the end of the sentence.

File: README.rst [hunk 4]
```diff
@@ -119,6 +129,55 @@ installation.
 
 Now the system is ready to use. For future sessions, only the step2 is required.
 
+Execution with dockers
+**********************
+Several dockers have been created to deploy the installation of one testbed or three
+glances. Even a docker for undeploying the testbed has been created.
+
+To create just one testbed, it is required, firstly to create the image and then to execute
```
Comment (flopezag) [inline README.rst]: you do not need to write to again before execute

File: README.rst [hunk 5]
```diff
@@ -119,6 +129,55 @@ installation.
...
+..code::
+    docker run -t delete_testbed -f docker/destroy_testbed/Dockerfile docker/destroy_testbed/
+    export OS_AUTH_URL = {the auth uri of the testbed against the tests are going to be execute}
+    export OS_USERNAME = {the user name}
+    export OS_TENANT_NAME = {the tenant name}
+    export OS_PASSWORD = {the password}
+    export OS_REGION_NAME = {the region}
+    export OS_PROJECT_DOMAIN_NAME = {the project domain name}
+    export OS_USER_DOMAIN_NAME = {the user domain name}
+    docker-compose -f docker/destroy_testbed/docker-compose.yml up
```
Comment (flopezag) [inline README.rst:177]: I assume that the export have to ve executed only one time, I mean it is common for the 3 docker execution commands (onetestbed, threetestbed, undeploy).

Comment (hmunfru) [inline README.rst:177]: normally they are not used at the same time, but in different moments

File: README.rst [hunk 6]
```diff
@@ -88,6 +88,15 @@ The micro flavor provides 64MB of RAM, 1 VCPU and 1GB of disk. The micro2 flavor
...
 
+Deploying three testbeds
+************************
+
+The *deploythreeglances.py* script deploys three Openstack Image Repositories (Glance) which
+shares the same Identity Service called Keystone. Each glance is deployed in a different server accesible by a
+floating IP. As well as previously, it is possible to access by SSH Key and its configuration
+variables are in the file *config_vars* in */home/ubuntu* folder inside the server.
+These three Glances testbed is used for GlanceSync acceptance tests.
+
```
Comment (flopezag) [inline README.rst:99]: The _deploythreeglances.py_ script deploys 3 OpenStack Image Repositories (Glance) which share the same Identity Service called Keystone. Each glance is deployed on a different server accesible by a floating IP. It is possible to access by SSH Key and its configuration variables are in the file _config_vars_ in _/home/ubuntu_ folder inside the server, in the same way that it was explained in "Deploying one testbed" section. These 3 Glance testbeds are used for GlanceSync acceptance tests.

File: README.rst (hunk 5 above)
Comment (flopezag) [inline README.rst:177]: NTC

**Comment by flopezag** [comment]
LGTM

**Pull request merged**